index.html

```diff
@@ -6254,6 +6257,32 @@ <h2>Acknowledgements</h2>
 Christopher Allen, Heather Vescent, and Wayne Chang.
     </p>
 
+    <p>
+The Working Group thanks the following individuals for their contributions to
+this specification (in alphabetical order): Denis Ah-Kang, Nacho Alamillo,
+Christopher Allen, Joe Andrieu, Antonio, Phil Archer, George Aristy, Baha, Juan
+Benet, BigBlueHat, Dan Bolser, Chris Boscolo, Pelle Braendgaard, Daniel Buchner,
+Daniel Burnett, By_caballero, Tim Cappalli, Melvin Carvalho, David Chadwick,
+Wayne Chang, Sam Curren, Hai Dang, Tim Daubenschütz, Oskar van Deventer, Kim
+Hamilton Duffy, Arnaud Durand, Ken Ebert, Veikko Eeva, Carson Farmer, Nikos
+Fotiou, Gabe, Gayan, Ryan Grant, Peter Grassberger, Adrian Gropper, Amy Guy,
+Daniel Hardman, Kyle Den Hartog, Philippe Le Hegaret, Ivan Herman, Michael
+Herman, Alen Horvat, Dave Huseby, Marcel Jackisch, Mike Jones, Andrew Jones, Tom
+Jones, jonnycrunch, Ted Thibodeau Jr, Gregg Kellogg, Michael Klein, Paul
+Knowles, David I. Lehn, Charles Lehner, Michael Lodder, Dave Longley, Tobias
+Looker, Wolf McNally, Robert Mitwicki, Mircea Nistor, Grant Noble, Mark
+Nottingham, Darrell O'Donnell, Vinod Panicker, Dirk Porsche, Praveen, Mike
+Prorock, Drummond Reed, Julian Reschke, Yancy Ribbens, Justin Richer, Rieks,
+Mikeal Rogers, Evstifeev Roman, Troy Ronda, Leonard Rosenthol, Michael Ruminer,
+Markus Sabadello, Cihan Saglam, Samu, Rob Sanderson, Wendy Seltzer, Mehran
+Shakeri, Jaehoon (Ace) Shim, Samuel Smith, James M Snell, SondreB, Manu Sporny,
+Orie Steele, Shigeya Suzuki, Sammotic Switchyarn, Oliver Terbu, Joel
+Thorstensson, Tralcan, Henry Tsai, Rod Vagg, Mike Varley, Kaliya "Identity
+Woman" Young, Eric Welton, Fuqiao Xue, @Yue, Dmitri Zagidulin, Brent Zundel,
+cabo, ewagner70, gimly-jack, gjgd, kdenhartog-sybil1, ktobich, mooreT1881, oare,
```

Comment:
```suggestion
Jones, jonnycrunch, Gregg Kellogg, Michael Klein, Paul Knowles, David I. Lehn,
Charles Lehner, Michael Lodder, Dave Longley, Tobias Looker, Wolf McNally,
Robert Mitwicki, Mircea Nistor, Grant Noble, Mark Nottingham, Darrell O'Donnell,
Vinod Panicker, Dirk Porsche, Praveen, Mike Prorock, Drummond Reed, Julian
Reschke, Yancy Ribbens, Justin Richer, Rieks, Mikeal Rogers, Evstifeev Roman,
Troy Ronda, Leonard Rosenthol, Michael Ruminer, Markus Sabadello, Cihan Saglam,
Samu, Rob Sanderson, Wendy Seltzer, Mehran Shakeri, Jaehoon (Ace) Shim, Samuel
Smith, James M Snell, SondreB, Manu Sporny, Orie Steele, Shigeya Suzuki,
Sammotic Switchyarn, Oliver Terbu, Ted Thibodeau Jr, Joel Thorstensson, Tralcan,
Henry Tsai, Rod Vagg, Mike Varley, Kaliya "Identity Woman" Young, Eric Welton,
Fuqiao Xue, @Yue, Dmitri Zagidulin, Brent Zundel, cabo, ewagner70, gimly-jack,
gjgd, kdenhartog-sybil1, ktobich, mooreT1881, oare,
```

Reply:
> removed the extra `kdenhartog-sybil1`

See reasoning for not removing the extra `kdenhartog-sybil1`: https://github.com/w3c/did-core/pull/780#discussion_r668709801

Reply:
Restored `kdenhartog-sybil1`.  Do what you will.